src/Accelerators/NNPA/Runtime/CMakeLists.txt

```diff
@@ -21,6 +21,6 @@ set_target_properties(RuntimeNNPA
   PROPERTIES
   LANGUAGE C
   POSITION_INDEPENDENT_CODE TRUE
-  COMPILE_OPTIONS "-O3;-fopenmp"
+  COMPILE_OPTIONS -option O3 fopenmp
```

Comment:
@tungld FYI  I added the -option to bypass this error I was seeing when compiling onnx-mlir locally.

`clang: error: unsupported option '-fopenmp'`

I got the idea to use option from the following documentation: https://cmake.org/cmake/help/latest/command/target_compile_options.html


Let me know if this is suitable.

Reply:
@chentong319 could you take a look at this? we are using clang by default to build onnx-mlir. Thanks! Since we haven't yet used openmp in zdnn extension, one way is to remove `-fopenmp`

Reply:
@gongsu832 also has a mechanism to invoke OpenMP only when it detects that OpenMP was built in the LLVM project. Maybe that is the mechanism to use here.

Reply:
@chentong319 said I can remove it for now